### PR TITLE
Fix dashes in obscured words

### PR DESCRIPTION
**Pull request by ecdye**

### CODE REVIEW

File: src/epub/text/chapter-34.xhtml [hunk 1]
```diff
@@ -52,10 +52,10 @@
...
+			<p>“Rosamond Oliver,” said he, “is about to be married to <abbr>Mr.</abbr> Granby, one of the best connected and most estimable residents in S⸺, grandson and heir to Sir Frederic Granby: I had the intelligence from her father yesterday.”</p>
 			<p>His sisters looked at each other and at me; we all three looked at him: he was serene as glass.</p>
 			<p>“The match must have been got up hastily,” said Diana: “they cannot have known each other long.”</p>
-			<p>“But two months: they met in October at the county ball at S-. But where there are no obstacles to a union, as in the present case, where the connection is in every point desirable, delays are unnecessary: they will be married as soon as S⁠—Place, which Sir Frederic gives up to them, can he refitted for their reception.”</p>
+			<p>“But two months: they met in October at the county ball at S⸺. But where there are no obstacles to a union, as in the present case, where the connection is in every point desirable, delays are unnecessary: they will be married as soon as S⁠⸺Place, which Sir Frederic gives up to them, can he refitted for their reception.”</p>
```
Comment (acabal) [inline src/epub/text/chapter-34.xhtml]: Good catch! Can you add a space after this one?

**Review by acabal — CHANGES_REQUESTED** [review]

File: src/epub/text/chapter-34.xhtml (hunk 1 above)
Comment (acabal) [reply src/epub/text/chapter-34.xhtml]: The instance of `S--Place`

Comment (ecdye) [reply src/epub/text/chapter-34.xhtml]: Yep, will do

Comment (ecdye) [reply src/epub/text/chapter-34.xhtml]: Done

**Comment by acabal** [comment]
Thanks!

**Pull request merged**